server/datastore/mysql/migrations/tables/20220428140039_CleanupEmptyHostEmails.go

```diff
@@ -0,0 +1,29 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220428140039, Down_20220428140039)
+}
+
+func Up_20220428140039(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+      DELETE FROM
+        host_emails
+      WHERE
+        email = ''`)
```

Comment:
Just for extra protection: `AND source = 'google_chrome_profiles'`, which is the source used to create emails from the Chrome profile osquery query.

Reply:
Well, actually we probably don't want empty emails regardless of source, nevermind that comment :D